pydata_center/monitoring/tests/test_helpers.py

```diff
@@ -135,3 +135,191 @@ def test_offline_logic_with_parameterized_cutoff(
 
         assert len(agents) == 1
         assert agents[0]['offline'] is expected_status
+
+    def setup_filtering_data(self):

```

Comment:
Rename `setup_filtering_data` → `setUpFilteringData` or` setUpData`
While not required, using camelCase for internal helpers in test classes is more conventional in Django test cases, and prefixing with setUp aligns with standard lifecycle methods.